app/main.ipynb

```diff
@@ -299,18 +397,29 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 17,
-   "outputs": [],
-   "source": [
-    "# write your code here"
-   ],
+   "source": "A[::2, :]",
```

Comment:
The selection of every second row is correct. Ensure that the slicing indices match the task requirements.